apps/vercel/src/app/install/service.tsx

```diff
@@ -16,8 +16,7 @@ export const registerOrganisation = async ({
   teamId,
   region,
 }: SetupOrganisationParams) => {
-  await getUsers(token, teamId);
-  const [organisation] = await db
+ await db
```

Comment:
call getUsers() here to check token validity as we don't habe an endpoint to check that.